jetson/ra_kinematics/mrover_arm.cpp

```diff
@@ -289,10 +289,11 @@ void StandardArm::target_orientation_callback(std::string channel, TargetOrienta
     plan_path(hypo_state, goal);
 }
 
-void MRoverArm::plan_path(ArmState& hypo_state, const std::vector<double> &goal) {
```

Comment:
You can keep this as const if you just pass original goal into the second call to rrt_connect